### ISSUE TITLE
Clear Line of Update Posting

### ISSUE THREAD

**Issue by Notsonoble** [Solaris-Skunk-Werks/solarisskunkwerks#68]
**Is your feature request related to a problem? Please describe.**
This is not a software feature, its a community "feature". With the 0.7.0 release we had double postings of threads because of no clear line of who's to put what where. Previous work on SSW had the same issue.

**Describe the solution you'd like**
A clean list of who posts updates and what updates should or shouldn't be posted where. Perhaps in the github wiki. If the group does this now it'll be less to struggle with in future.

**Describe alternatives you've considered**
The alternatives are what we have now. Someone notices an update and people go posting wherever they normally participate in the bt community whole.

Or one person is responsible for all of it, from marking a release clear to posting about it in every place people discuss battletech.

**Additional context**
I think SSW's kinda died its two previous deaths because in both situations it was a "single person project", something simple like "hey you are responsible for making sure these people know" would help it not feel like an all on the one shoulders burden.

**Comment by WEKarnesky** [comment]
Agreed on all accounts.  I am open to input on how best to organize.

Let's start by using this to create a list of tasks and sub-tasks to delegate out.

**Comment by Notsonoble** [comment]
So I'll start a wiki page...

**Comment by Notsonoble** [comment]
https://github.com/Solaris-Skunk-Werks/solarisskunkwerks/wiki/Update-Announcements created. A similar page (or continuation of discussion here) could be used to continue organization.

**Comment by Maelwys** [comment]
http://www.solarisskunkwerks.com/

Either Skyhigh or I should be able to update the site if need be. 

There's also a slack group that has github integration, and there's also a Dropbox account that hosts the Mech files and can host the beta to make sure that changes aren't going to break the mech files too much.

**Comment by WEKarnesky** [comment]
> http://www.solarisskunkwerks.com/
> 
> Either Skyhigh or I should be able to update the site if need be.
> 

That would be great, I spoke to PorterPirate months ago and access wasn't possible at the time, he no longer had access to log into the website

> There's also a slack group that has github integration, and there's also a Dropbox account that hosts the Mech files and can host the beta to make sure that changes aren't going to break the mech files too much.

We've mostly been working on Discord, but we will happily accept invites, we had no clue there was a slack group

**Comment by WEKarnesky** [comment]
@Notsonoble Do we consider this resolved?
